### PR TITLE
Add training orchestration tool

### PR DESCRIPTION
**Pull request by zhimin-z**
Add Nebullvm, and remove one of its specific components - Speedster.

### CODE REVIEW

**Comment by axsaucedo** [comment]
This file was recently changed via https://github.com/EthicalML/awesome-production-machine-learning/pull/305/files by @emilecourthoud , I suggest getting further alignment via an issue (or thought discussion on this PR to align on whether we would keep it as NevuLLM vs Speedster, are there any strong opinions? I will close for the time being but based on decision we can reopen

File: README.md [hunk 1]
```diff
@@ -418,7 +418,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [H2O-3](https://github.com/h2oai/h2o-3) ![](https://img.shields.io/github/stars/h2oai/h2o-3.svg?style=social) - Fast scalable Machine Learning platform for smarter applications: Deep Learning, Gradient Boosting & XGBoost, Random Forest, Generalized Linear Modeling (Logistic Regression, Elastic Net), K-Means, PCA, Stacked Ensembles, Automatic Machine Learning (AutoML), etc..
 * [Jax](https://github.com/google/jax) ![](https://img.shields.io/github/stars/google/jax.svg?style=social) - Composable transformations of Python+NumPy programs: differentiate, vectorize, JIT to GPU/TPU, and more.
 * [Modin](https://github.com/modin-project/modin) ![](https://img.shields.io/github/stars/modin-project/modin.svg?style=social) - Speed up your Pandas workflows by changing a single line of code.
-* [Speedster](https://github.com/nebuly-ai/nebullvm/tree/main/apps/accelerate/speedster) ![](https://img.shields.io/github/stars/nebuly-ai/nebullvm.svg?style=social) - Automatically apply SOTA optimization techniques to achieve the maximum inference speed-up on your hardware.
```
Comment (zhimin-z) [inline README.md:421]: Also, S-letter (Speedster) should not appear before N-letter (Numba), @axsaucedo what do you think?

Comment (axsaucedo) [reply README.md:421]: Yes that is correct, good catch

**Comment by axsaucedo** [comment]
I'm still not clear on why naming conventions are being switched back and fort but we can merge for now

**Pull request merged**